### PR TITLE
fix: message button double-click unselected text

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What type of PR is this?

- [ ] Feature
- [x] Bugfix
- [ ] Test
- [ ] Documentation
- [ ] Refactor
- [ ] Style
- [ ] Chore

## Which issue(s) this PR fixes:

Fixes # message button double-click unselected text

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [x] No

### CODE REVIEW

File: shell/app/layout/pages/page-container/components/sidebar.scss [hunk 1]
```diff
@@ -58,3 +58,9 @@
     }
   }
 }
+
+.message-badge {
+  -moz-user-select: none;
+  -webkit-user-select: none;
+  user-select: none;
+}
```
Comment (daskyrk) [inline shell/app/layout/pages/page-container/components/sidebar.scss:66]: use `unselectable` in util.scss

**Review by laojun — APPROVED** [review]

**Pull request merged**